### DISCUSSION TITLE
Refund Request

### DISCUSSION THREAD

**Discussion by D4VSZ** [aclap-dev/video-downloadhelper#1055] [License & Purchase requests]
### What's your issue?

Hello, I recently purchased a license for Chrome and saw that it does not support Youtube. I would like to cancel my license so that I can purchase the Firefox version. Thanks

### The 10 first digits (only 10) of your license number or email address

324a88efba

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
Take a look at #1054.

**Comment by D4VSZ** [comment]
I purchased the key via another email and I am currently waiting for it to be cancelled.

**Reply by jcfrog** [reply]
Hi,
Sorry I missed your message.
You have been refunded. Should be available in a few business days.

**Comment by Pivokast** [comment]
Same as first comment, I started using this because it was on chrome but now you can't use it in chrome, so can you either give me the code for the Firefox browser or refund the purchase, thank you.

My ten digit number is: 6afd06ee2c

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by LordPlay23** [comment]
i started using this because i wanted it for youtube but it dosent work for that ....Thank you

9cb1681fe

**Reply by jcfrog** [reply]
Hi,
You have been refunded. Should be available in a few business days.
